app/_src/mesh/features/meshopa.md

```diff
@@ -0,0 +1,668 @@
+---
+title: MeshOPA (beta) - OPA Policy Integration
+---
+
+{% warning %}
+This policy uses new policy matching algorithm and is in beta state,
+it should not be mixed with [OPA Policy](../opa).
+{% endwarning %}
+
+## MeshOPA policy plugin
+
+{{site.mesh_product_name}} integrates the [Open Policy Agent (OPA)](https://www.openpolicyagent.org/) to provide access control for your services.
+
+The agent is included in the data plane proxy sidecar, instead of the more common deployment as a separate sidecar.
+
+When `MeshOPA` policy is applied, the control plane configures:
+
+- The embedded policy agent, with the specified policy
+- Envoy, to use [External Authorization](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/ext_authz/v3/ext_authz.proto) that points to the embedded policy agent
+
+## TargetRef support matrix
+
+| TargetRef type    | top level | to  | from |
+| ----------------- | --------- | --- | ---- |
+| Mesh              | ✅        | ❌  | ❌   |
+| MeshSubset        | ✅        | ❌  | ❌   |
+| MeshService       | ✅        | ❌  | ❌   |
+| MeshServiceSubset | ✅        | ❌  | ❌   |
+| MeshGatewayRoute  | ❌        | ❌  | ❌   |
+
+To learn more about the information in this table, see the [matching docs](/docs/{{ page.version }}/policies/targetref).
+
+## Configuration
+
+To apply a policy with MeshOPA:
+
+- Specify the group of data plane proxies to apply the policy to with the `targetRef` property.
+- Provide the list of policies<a href="#footnote-1"><sup id="note-return-1">1</sup></a> with the `appendPolicies` property. Policies are defined in the [Rego language](https://www.openpolicyagent.org/docs/latest/policy-language/).
+- Optionally provide custom configuration for the policy agent.
+
+<span id="footnote-1"><a href="#note-return-1">1</a></span>: due to a [bug in OPA](https://github.com/open-policy-agent/opa/issues/5595) we currently support only one policy. 
```

Comment:
`<span id="footnote-1"><a href="#note-return-1">1</a></span>` why we need this clickable `1`?

Reply:
also can this be changed to some sort of warning or time that people won't miss it when reading this docs?

Reply:
> why we need this clickable 1?

I just followed the [best practices](https://github.com/Kong/docs.konghq.com/blame/3e0557f2d96725ebddbb6a3e6300fdf9a36f61bf/archive/studio/old/1.2.x/best-practices.md#L45) for footnotes, but it does not really matter if we apply @cloudjumpercat 

> also can this be changed to some sort of warning or time that people won't miss it when reading this docs?

is "**Note:**" (as Diana suggested) ok?

Reply:
not applicable any more, closing